core/deployment/src/main/java/io/quarkiverse/langchain4j/deployment/AiServicesProcessor.java

```diff
@@ -1,57 +1,5 @@
 package io.quarkiverse.langchain4j.deployment;
 
-import static dev.langchain4j.exception.IllegalConfigurationException.illegalConfiguration;
```

Comment:
Not sure what happened with the import related changes...

Reply:
Don't worry, I did not run the formatter